### DISCUSSION TITLE
Integration failed - Model not supported?

### DISCUSSION THREAD

**Discussion by mrbreaks90** [pail23/stiebel_eltron_isg_component#167] [Q&A]
Hi! First of all - thank you for your work! I tried to integrate our WPE-I 33 H 400 Premium, but I get an error message. Serviceportal is working, IP adress is double-checked. Do you have an idea if I'm doing something wrong? Thank you in advance. Error log is attached.
Best regards
[home-assistant_stiebel_eltron_isg_2024-01-06T23-38-10.562Z.log](https://github.com/pail23/stiebel_eltron_isg_component/files/13851679/home-assistant_stiebel_eltron_isg_2024-01-06T23-38-10.562Z.log)

**Comment by mrbreaks90** [comment]
I will allow myself to follow up again. An integration would be extremely valuable, as it would allow me to record the sensor data over time and fine-tune the heat pump.

Unfortunately, I'm not particularly well versed. So thank you in advance for any tips!

**Comment by Optic00** [comment]
WPM G and WPE-I support is not available today. The scope and architecture decision for that hardware family is now tracked centrally in issue #684. Closing this discussion in favor of #684. Please follow and comment there.
